### PR TITLE
Add colored edges to the cards

### PR DESCRIPTION
**Pull request by fkz**
Based on the type, add colored edges. The colors are picked somewhat randomly for now:
- yellow for actions
- blue for items
- green for spells

Test print:

### CODE REVIEW

File: card-style.cls [hunk 1]
```diff
@@ -36,6 +37,27 @@
...
+      \draw [color=\cardcolor,line width=2mm]
+        (current page.south west)
+        rectangle
+        (current page.north east);
+    }
+  \fi
+}
+
+\xdef\CurrentCardCategory{None}
+\AddToHook {shipout/background}{\CardType{\CurrentCardCategory}}
```
Comment (Toxaris) [inline card-style.cls]: I'm confused about the words "category" vs "type" here.

File: card-style.cls [hunk 2]
```diff
@@ -36,6 +37,27 @@
...
+  \ifnum\pdfstrcmp{#1}{Focus}=0     \def\cardcolor{green} \fi
+  
+  \ifnum\pdfstrcmp{\cardcolor}{}=0 \else
+    \tikz[remember picture,overlay] {%
+      \draw [color=\cardcolor,line width=2mm]
+        (current page.south west)
+        rectangle
+        (current page.north east);
+    }
+  \fi
```
Comment (Toxaris) [inline card-style.cls]: round corners?

File: card-style.cls [hunk 3]
```diff
@@ -36,6 +37,27 @@
 
 \newcounter{cards}
 
+\newcommand{\CardType}[1]{%
+  \def\cardcolor{}
+  \ifnum\pdfstrcmp{#1}{Item}=0      \def\cardcolor{blue} \fi
+  \ifnum\pdfstrcmp{#1}{Action}=0    \def\cardcolor{yellow} \fi
+  \ifnum\pdfstrcmp{#1}{Spell}=0     \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Cantrip}=0   \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Focus}=0     \def\cardcolor{green} \fi
```
Comment (Toxaris) [inline card-style.cls]: This looks like something `pgfkeys` can do for us, similar to `every ...` in tikz. Along these lines:

```latex
\tikzset{card background=line width=2mm}
\tikzset{Item card background=card background, color=blue}
...

\newcommand{\CardType}[1]{%
  \tikz[remember picture, overlay]{%
    \draw[#1 card background]
      ...
```

File: card-style.cls [hunk 4]
```diff
@@ -36,6 +37,27 @@
 
 \newcounter{cards}
 
+\newcommand{\CardType}[1]{%
+  \def\cardcolor{}
+  \ifnum\pdfstrcmp{#1}{Item}=0      \def\cardcolor{blue} \fi
```
Comment (Toxaris) [inline card-style.cls]: somehow I was assuming we distinguish between permanent and consumable items

**Review by Toxaris — COMMENTED** [review]
Some thoughts.

File: card-style.cls (hunk 3 above)
Comment (fkz) [reply card-style.cls]: Oh, yes that looks nice

File: card-style.cls (hunk 2 above)
Comment (fkz) [reply card-style.cls]: Yes, I was still in the phase of getting something working at all 😃

File: card-style.cls (hunk 1 above)
Comment (fkz) [reply card-style.cls]: Indeed, we should use more consistent phrasing.

**Comment by fkz** [comment]
@Toxaris Feel free to work further on this, I'll stop working on it for now.

File: card-style.cls (hunk 4 above)
Comment (fkz) [reply card-style.cls]: 🤷

**Comment by fkz** [comment]
Ah, ich war die ganze Zeit verwirrt, warum hier der latex-Check nicht kommt, aber das liegt daran, dass der PR von nem Fork ausgeht und deswegen nicht die Berechtigung dazu hat. Und die Fehlerbehandlung ist dann so, dass alles grün ist.

**Pull request merged**